.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
.idea is meant for the global git ignore, and not the local project git ignore's.